gradle/libs.versions.toml

```diff
@@ -0,0 +1,33 @@
+[versions]
+kotlin = "1.8.0"
+openrndr = "0.4.2"
```

Comment:
I feel it would be a bit backwards to explain how to use a snapshot without explaining how to create one. And if we were to do both, we might as well point users to this link https://github.com/openrndr/orx#publish-and-use-local-builds-of-the-library-in-your-applications

Reply:
What do you think is better? To leave that link there as a comment? Or no comments?